drivers/hyperv/hyperv.go

```diff
@@ -91,6 +91,10 @@ func (d *Driver) SetConfigFromFlags(flags drivers.DriverOptions) error {
 	return nil
 }
 
+func toBytes(gibibytes uint) string {
+	return fmt.Sprintf("%d", gibibytes*1024*1024*1024)
```

Comment:
Gibi like the Shadoks :) ? https://en.wikipedia.org/wiki/Les_Shadoks

Reply:
I did not name the unit :) https://en.wikipedia.org/wiki/Gibibyte 
Gigabytes probably make as much sense as Gibibytes in this context, I can change it.

Reply:
this sounds like a repeat of what we did for Minishift ;-)